### PR TITLE
chore: removed Jest types and fixed tests

### PR DESCRIPTION
**Pull request by gentlementlegen**
Follows up comment https://github.com/ubiquity/work.ubq.fi/pull/25#issuecomment-1980128894

### CODE REVIEW

File: .github/workflows/cypress-testing.yml [hunk 1]
```diff
@@ -24,7 +22,7 @@ jobs:
           CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           # Values are set because the code requires them, but they are not actually used.
-          SUPABASE_URL: "https://wfzpewmlyiozupulbuur.supabase.co"
+          SUPABASE_URL: "https://iyybhhiflwbsjopsgaow.supabase.co"
```
Comment (0x4007) [inline .github/workflows/cypress-testing.yml]: Shouldn't we revert this URL?

**Comment by 0x4007** [comment]
https://github.com/pavlovcik/work.ubq.fi/actions/runs/8181893746/job/22377222764?pr=1#step:4:99

I thought these are warnings not errors. I'm not sure what to do here aside from the deep dive of debugging it, perhaps on another task?

**Comment by gentlementlegen** [comment]
> https://github.com/pavlovcik/work.ubq.fi/actions/runs/8181893746/job/22377222764?pr=1#step:4:99
> 
> I thought these are warnings not errors. I'm not sure what to do here aside from the deep dive of debugging it, perhaps on another task?

It is weird because the real API should never be hit, which is the point of the mocks. I tried and I don't seem to have this happening locally so I am a bit confused. I will give it a deeper look. But this happens when a task detail is fetched through the `octokit` module.

**Comment by 0x4007** [comment]
> > https://github.com/pavlovcik/work.ubq.fi/actions/runs/8181893746/job/22377222764?pr=1#step:4:99
> > I thought these are warnings not errors. I'm not sure what to do here aside from the deep dive of debugging it, perhaps on another task?
> 
> It is weird because the real API should never be hit, which is the point of the mocks. I tried and I don't seem to have this happening locally so I am a bit confused. I will give it a deeper look. But this happens when a task detail is fetched through the `octokit` module.

I forget why you are adverse to doing a real e2e test in our CI. If its just for speed maybe it doesn't really matter much?

File: .github/workflows/cypress-testing.yml (hunk 1 above)
Comment (0x4007) [reply .github/workflows/cypress-testing.yml]: This can be merged when you address this @FernandVEYRIER

**Comment by gentlementlegen** [comment]
> > > https://github.com/pavlovcik/work.ubq.fi/actions/runs/8181893746/job/22377222764?pr=1#step:4:99
> > > I thought these are warnings not errors. I'm not sure what to do here aside from the deep dive of debugging it, perhaps on another task?
> > 
> > 
> > It is weird because the real API should never be hit, which is the point of the mocks. I tried and I don't seem to have this happening locally so I am a bit confused. I will give it a deeper look. But this happens when a task detail is fetched through the `octokit` module.
> 
> I forget why you are adverse to doing a real e2e test in our CI. If its just for speed maybe it doesn't really matter much?

Nothing against a few real test case uses, but they should be sparsely used as they are not controllable. And for the same reason they make test flaky as the expected result is always changing (e.g. we open or close new tickets, so the ticket list is gonna be different from test expectations).

On this regard actually someone created a mocking library that it might be interesting looking into
https://github.com/Chocrates/octomock

**Comment by gentlementlegen** [comment]
@pavlovcik you should be good to go. Actually Supabase changed their login urls which was one cause of the test failure.

File: cypress/e2e/devpool.cy.ts [hunk 2]
```diff
@@ -132,7 +132,7 @@ describe("DevPool", () => {
         body: githubUser,
       });
     }).as("getUser");
-    cy.intercept("https://iyybhhiflwbsjopsgaow.supabase.co/auth/v1/authorize?provider=github", (req) => {
+    cy.intercept("https://github.com/login/oauth/authorize**", (req) => {
```
Comment (0x4007) [inline cypress/e2e/devpool.cy.ts:135]: Not familiar with this but okay.

**Review by 0x4007 — APPROVED** [review]

**Comment by 0x4007** [comment]
I think there's some instability with the new private issues

**Comment by 0x4007** [comment]
Oh @FernandVEYRIER you aimed this at the wrong repo. This is my fork. You need to merge to ubiquity 

I would but I'm not on my computer.

**Comment by gentlementlegen** [comment]
@pavlovcik I thought we would merge my branch into yours, then yours into the base repository

**Comment by 0x4007** [comment]
> take over and merge 

https://github.com/ubiquity/work.ubq.fi/pull/25#issuecomment-1980128894

And the assignment to you meant that you can push to my pull and merge it lol

I'm just a bit in a weird limbo state where I shouldn't be coding but I can't resist dipping in and out briefly for small things.

**Comment by gentlementlegen** [comment]
> > take over and merge
> 
> [ubiquity#25 (comment)](https://github.com/ubiquity/work.ubq.fi/pull/25#issuecomment-1980128894)
> 
> And the assignment to you meant that you can push to my pull and merge it lol
> 
> I'm just a bit in a weird limbo state where I shouldn't be coding but I can't resist dipping in and out briefly for small things.

I overthought it I guess haha, no worries

**Pull request merged**